packages/fxa-content-server/app/scripts/views/settings/avatar_crop.js

```diff
@@ -49,36 +46,36 @@ const View = FormView.extend({
     var height = this._cropImg.get('height');
     var src = this._cropImg.get('src');
 
-    try {
-      this.cropper = new Cropper({
-        container: this.$('.cropper'),
-        displayLength: Constants.PROFILE_IMAGE_DISPLAY_SIZE,
-        exportLength: Constants.PROFILE_IMAGE_EXPORT_SIZE,
-        height: height,
-        horizontalGutter: HORIZONTAL_GUTTER,
-        onRotate: this._onRotate.bind(this),
-        onTranslate: this._onTranslate.bind(this),
-        onZoomIn: this._onZoomIn.bind(this),
-        onZoomOut: this._onZoomOut.bind(this),
-        onZoomRangeChange: this._onZoomRangeChange.bind(this),
-        src: src,
-        verticalGutter: VERTICAL_GUTTER,
-        width: width,
-      });
-    } catch (e) {
-      // settings_common functional tests visit this page directly so draggable
-      // won't be preloaded. Ignore errors about that– they don't matter.
-      if (
-        this.broker.isAutomatedBrowser() &&
-        e.message.indexOf('draggable') !== -1
-      ) {
-        return;
+    setTimeout(() => {
```

Comment:
This was critical in effectively calculating the cropper wrapper dimensions, but I'd like to remove it because it feels gross. I'm not sure why this issue is occurring, though, since `afterVisible` is being called after the elements are rendered. Any suggestions here are appreciated.